drivers/bluetooth/hci/Kconfig.nxp

```diff
@@ -0,0 +1,18 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+if BT_NXP_CTLR
+
+config BT_NXP_NW612
+	bool "NXP IW612 Chipset"
+	help
+		NXP IW612 Chipset supports Wi-Fi® 802.11a/b/g/n/ac/ax + Bluetooth® 5.3
+		BR/EDR/LE + IEEE802.1.5.4 up to 601 Mbps data rate on Wi-Fi® and 2Mbps
+		data rate on Bluetooth®. 4-wire UART@3M baud is supported. PCM for
+		audio is also supported.
+		Details of the module could be fond on https://www.nxp.com/products/
+		wireless-connectivity/wi-fi-plus-bluetooth-plus-802-15-4/2-4-5-ghz-
+		dual-band-1x1-wi-fi-6-802-11ax-plus-bluetooth-5-4-plus-802-15-4-tri-
+		radio-solution:IW612.
```

Comment:
Nit: indent help with a tab and 2 spaces

Reply:
Fixed.